### ISSUE TITLE
OpenTelemetry Swift SDK as submodule

### ISSUE THREAD

**Issue by swasti29** [open-telemetry/opentelemetry-swift#341]
Hi Team,

Can i add Opentelemetry swift sdk as submodule to my SDK?

**Comment by nachoBonafonte** [comment]
Yes, I would say most users users are using it like that. You can add the products needed from the opentelemetry-swift project adding it to your Project ( minimally OpenTelemetryApi and OpenTelemetrySdk) add the dependencies both in Xcode and SPM projects as you would do with any other SPM library. They link statically by default so you can link it with your SDK without problems

**Comment by swasti29** [comment]
@nachoBonafonte Can i use this library as souce without swift manager? Can i just add it as git submodule?

**Comment by nachoBonafonte** [comment]
I have never tried it, but I would say so.  It is just code that you can compile and link like any other swift file.
